lib/src/solana_wallet.dart

```diff
@@ -1,35 +1,58 @@
 import 'package:bip39/bip39.dart' as bip39;
 import 'package:cryptography/cryptography.dart';
 import 'package:solana/src/base58/base58.dart' as base58;
+import 'package:solana/src/types/hd_key_pair.dart';
 
 /// Signs transactions to be sent to a Solana Cluster
 class SolanaWallet {
-  SolanaWallet._fromKeyPair(KeyPair keyPair, this.address) : _keyPair = keyPair;
-
-  static final _ed25519 = Ed25519();
+  SolanaWallet._fromKeyPair(
+    this._keyPair, {
+    required this.address,
+  });
 
   /// Creates and initializes a new SolanaWallet for the given bip39
   /// [mnemonic] string of 12 words
-  static Future<SolanaWallet> fromMnemonic(String mnemonic) async {
+  static Future<SolanaWallet> fromMnemonic(
+    String mnemonic, {
+    int? account,
```

Comment:
Please add info about this parameter to the docstring. It's very confusing otherwise.

Reply:
Agree!